layouts/picker.json

```diff
@@ -524,7 +524,7 @@
   },
   {
     "label": "Controls",
-    "cols": 4,
+    "cols": 5,
```

Comment:
I believe @jacobgkau is referring to this change. What's the reasoning for it?

Reply:
It didn't align the button correctly without making that change. It would be like this:

| Screen Up | Screen Down |
| Suspend |
| Power |

Instead of 

| Screen Up | Screen Down |
| Suspend | Power |

Reply:
@ids1024 any thoughts on this? It seems weird to make a change that affects the laptop keyboard control button group in order to better arrange the Launch control button group. It doesn't seem to break anything though, so I'm not sure I have a huge problem with it.

Reply:
@ahoneybun can you remove this change? It should be fixed by https://github.com/pop-os/keyboard-configurator/pull/98

Once that's done, I'm ready to approve this PR.

Reply:
Should be changed now though we need @ids1024 to review it again now.

Reply:
Thanks! I'll have one last look at it soon.